rotkehlchen/tests/api/test_trades.py

```diff
@@ -259,6 +259,34 @@ def test_add_trades(rotkehlchen_api_server):
     assert data['message'] == ''
     assert data['result']['entries'] == [{'entry': new_trade, 'ignored_in_accounting': False}]
 
+    # Test trade with rate 0. Should fail
+
+    zero_rate_trade = {
+        'timestamp': 1575640208,
+        'location': 'external',
+        'pair': 'ETH_WETH',
+        'trade_type': 'buy',
+        'amount': '0.5541',
+        'rate': '0',
+        'fee': '0.01',
+        'fee_currency': 'USD',
+        'link': 'optional trader identifier',
+        'notes': 'optional notes',
+    }
+
```

Comment:
nitpick: don't put so many empty lines. There is no reason for 263, 276, 283 to be empty lines. No need to have each statement between empty lines.